src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -380,6 +380,19 @@ class ContentExplorer extends Component {
         return <ContentExplorerEmptyState isSearch={isViewingSearchResults} />;
     };
 
+    getUpdatedChooseButtonProps = chooseButtonProps => {
+        const { selectedItems } = this.state;
+        const { contentExplorerMode } = this.props;
+        const chosenItems = getChosenItemsFromSelectedItems(selectedItems);
+        if (contentExplorerMode === ContentExplorerModes.SELECT_FOLDER && chosenItems.length === 0) {
+            return {
+                ...chooseButtonProps,
+                isDisabled: chooseButtonProps.isDisabled && this.getCurrentFolder().id === '0',
```

Comment:
There could be other use cases where the consumer of this ContentExplorer feature would want to be able to select the root folder. Is there a way to make use of the `areButtonsDisabled` field instead, from within the repo where you are using this component? If the root folder has `isActionDisabled` set to true, then it should disable the choose button when items are deselected:
https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/ContentExplorer.js#L446-L450
https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js#L108

Reply:
However in this case when `this.getCurrentFolder().id === '0'`  File name will be **"All files"**  it means there is no root folder can be selected for user?  Try to set root folder has `isActionDisabled` to true however it still get overwritten by the chooseButtonProps.